metadata/boundaries/county_boundaries

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah County Boundaries
+
+## ID
+
+b2c8af56-3d92-4aa7-ac2f-b1388051d9ce
+
+## Brief Summary
+
+A statewide polygon dataset of county boundaries for cartography and approximate boundary identification.
+
+## Summary
+
+This dataset contains polygons for all 29 counties in Utah, where the perimeter of the polygon represents the county boundary. Each polygon also contains county population as of the [2020 Census](https://www.census.gov/library/stories/state-by-state/utah-population-change-between-census-decade.html) and current approximate population.
```

Comment:
```suggestion
This dataset contains polygons for all 29 counties in Utah, where the perimeter of the polygon represents the county boundary. Each polygon also contains county population as of the [2020 Census](https://www.census.gov/library/stories/state-by-state/utah-population-change-between-census-decade.html) and the current approximate population.
```

Current pop estimate as of when? Did we figure out when we update it and from where?

Reply:
@gregbunce @DenisePeterson  The Municipalities dataset has a population estimate field as well, and that one is as of July 2022. Is that true of the Counties data too?